microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -225,7 +225,8 @@ Header Parameters: Authorization: Bearer $TOKEN
 > curl \
 -H "Authorization: Bearer $TOKEN" \
 -X GET \
--v \ https://api.addons.microsoftedge.microsoft.com/v1/products/$productID/submissions/operations/{operationID}
+-v \
+https://api.addons.microsoftedge.microsoft.com/v1/products/$productID/submissions/operations/{operationID}
```

Comment:
I don't have a way to test this, is the format change correct?

Reply:
Yes @vchapel. I have verified it.